third_party/move/move-core/types/src/language_storage.rs

```diff
@@ -108,7 +108,7 @@ pub struct StructTag {
     pub module: Identifier,
     pub name: Identifier,
     // alias for compatibility with old json serialized data.
-    #[serde(rename = "type_args", alias = "type_params")]
+    #[serde(rename = "type_args")]
```

Comment:
It's interesting how this causes issues for this test, though I'm not sure what the implications are

Reply:
As I said on slack, its unclear to me why this is needed and what are the implications. Why doesn't this work any longer? Bug in serde?